examples/tv-app/android/java/ChannelManager.cpp

```diff
@@ -232,6 +232,14 @@ CHIP_ERROR ChannelManager::HandleGetCurrentChannel(AttributeValueEncoder & aEnco
                 channelInfo.affiliateCallSign = Optional<CharSpan>(affiliateCallSign.charSpan());
             }
 
+            jfieldID getJidentifierField = env->GetFieldID(channelClass, "identifier", "Ljava/lang/String;");
```

Comment:
not about this PR, just take a look at this callback function, channelInfoList is potentially leaking with local object reference. 
Could you put the below in the beginning of this HandleGetCurrentChannel callback function so that we can automatically release local reference when it is out of scope.
    JNIEnv * env   = JniReferences::GetInstance().GetEnvForCurrentThread();
    VerifyOrReturn(env != nullptr, ChipLogError(Controller, "Could not get JNIEnv for current thread"));
    **JniLocalReferenceManager manager(env);**
Thanks

Reply:
fixed - thanks!